aws-replicator/aws_replicator/client/auth_proxy.py

```diff
@@ -346,7 +349,13 @@ def start_aws_auth_proxy_in_container(
     ]
     env_vars = env_vars or os.environ
     env_vars = select_attributes(dict(env_vars), env_var_names)
-    env_vars["LOCALSTACK_HOST"] = "host.docker.internal"
```

Comment:
Note that this doesn't work in some Linux environments - we now leverage the `get_docker_host_from_container()` utility further below, which internally also returns `host.docker.internal` **if** that host resolves / is accessible.